demo/apisix/tmail-apisix-plugin-runner/src/main/java/com/linagora/apisix/plugin/RedisRevokedTokenRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.linagora.apisix.plugin;
+
+import java.time.Duration;
+
+import org.springframework.data.redis.core.StringRedisTemplate;
+
+public class RedisRevokedTokenRepository implements IRevokedTokenRepository {
+    public static final Duration CACHE_DURATION = Duration.ofMinutes(60);
```

Comment:
This duration is likely too short and should exceed refresh token validity to NOT expose us to security weaknesses.


I propose 1 week as a minimum...